### PR TITLE
Prepare for release 2.3.5

### PR DESCRIPTION
**Pull request by moon0326**
### Changes proposed in this Pull Request:

Preparation for release of 2.3.5

### Changelog

```
= 2.3.5 =
* Add a new class to customize for Square from Partner Aware Onboarding #1426
* Remove the Customizer from the admin menu and the admin bar, if a block theme is used #1433
* Delete woocommerce_demo_store option to hide the demo store notice #1433
```

- https://github.com/Automattic/wc-calypso-bridge/pull/1426#top
- https://github.com/Automattic/wc-calypso-bridge/pull/1433#top

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -22,6 +22,9 @@ This section describes how to install the plugin and get it working.
 
 == Changelog ==
 
+= 2.3.5 =
+* _Enter your changes here_
```
Comment (ilyasfoo) [inline readme.txt:26]: Replace this with the changes under `unreleased`

**Review by ilyasfoo — CHANGES_REQUESTED** [review]
1 comment to update changelog but otherwise looking good!

File: readme.txt (hunk 1 above)
Comment (moon0326) [reply readme.txt]: Thank you for the catch. I forgot to bring that commit back from the previous one 😄 

Updated it in https://github.com/Automattic/wc-calypso-bridge/pull/1438/commits/971d9a41676e0b7d185599caa7432c19fbfe4631

**Review by ilyasfoo — APPROVED** [review]
Thanks! LGTM 🚢

**Review by PanosSynetos — COMMENTED** [review]
Bumped versions looking good. Thanks!

**Pull request merged**